addons/website/static/src/scss/website.scss

```diff
@@ -151,6 +151,10 @@ $-seen-urls: ();
         $custom-colors: append($custom-colors, $key);
     }
     @include print-variable('custom-colors', $custom-colors);
+
+    //7) Set colors for the drop zone
+    @include print-variable('dropzone-contrast-rgb', to-rgb(color-contrast($body-bg, $color-contrast-dark: darken($o-we-handles-accent-color, 25%))));
+    @include print-variable('dropzone-bg-color-rgb', to-rgb(mix($o-we-handles-accent-color, $body-bg, 50%)));
```

Comment:
- `website.scss` is a visitor SCSS file, not an edit mode one. Even though most of the variable above are only used for edit mode, they technically make sense to be available for visitor pages. Those 2 new ones do not: they should only be available in edit mode SCSS files, in fact probably in the same place where you use them? `website.edit_mode.scss`?

- Although, why is it a website code? Should not it be in web_editor so that mass_mailing have the same design? I tested and there are indeed differences: already when you just drag and drop a snippet from the right panel.

- Speaking of dropdown, this is the only thing that I don't really like in the new design (I know @sobo-odoo had the same opinion but did not dare saying it out loud here :smile: Yes I changed my mind when seeing it in action @sobo-odoo). I don't mind removing the border and having a blurry effect, but they are multiple problems with those I think:
    - We don't see them as well, especially since they entirely disappear from the screen with their animation
    - When they go across each other, it's messy (always the case with inner snippets)
    - There is a strange (IMO) margin on both sides
    - Test drag and dropping fields in a "form" snippet -> this is broken :wink: 
    - Is there really a reason to have 2 kind of drop-zones in the end? :)

Reply:
I agree that the new inner dropzone are not visible enough + tends to overlap too much.
Regarding the "section" dropzone, I think I prefer when they were more full width.
cc @qsm-odoo